.eslintrc

```diff
@@ -42,7 +42,7 @@
           ["^react", "^@?\\w"],
           // Internal packages.
           [
-            "^(components|constants|data|hooks|i18n|images|reducers|sagas|types|utils|validations|views)(/.*|$)"
+            "^(components|constants|contexts|data|hooks|i18n|images|reducers|sagas|types|utils|validations|views)(/.*|$)"
```

Comment:
We need to add `contexts` here so ESLint knows that this is an internal directory and not an NPM import. This is important so it knows how to auto sort our imports.

Because of this tweak in the rule, there will be a handful of files that just fix the ordering of the imports.